src/polytopes/hexahedron.jl

```diff
@@ -9,6 +9,8 @@ A hexahedron with points `p1`, `p2`, ..., `p8`.
 """
 @polytope Hexahedron 3 8
 
+lentype(::Type{<:Hexahedron{Dim,P}}) where {Dim,P} = lentype(P)
```

Comment:
the type P could be stored in the abstract Polytope type, and the method lentype could be defined for all polytopes at once.